s32/CMakeLists.txt

```diff
@@ -24,6 +24,9 @@ if(CONFIG_HAS_MCUX)
   #
   # Glue code must be added for each device supported to build with MCUX. This
   # can be found in hal_nxp/s32/mcux/devices:
+  # - s32/mcux/devices/${MCUX_DEVICE}/${MCUX_DEVICE}_device.h
+  #   redefine RTD module's peripheral access layers and its register masks to be
+  #   compatible with MCUX drivers for incompatible peripherals (FlexCAN)
```

Comment:
just a minor comment, please remove (FlexCAN) since this is intended to be a generic description. We may need to do the same for other peripherals as well.

Reply:
Okay, I updated. I thought that need to note specific peripherals because it is not all peripherals need to be done.